cmd/ketch/cname_add.go

```diff
@@ -31,6 +33,8 @@ func newCnameAddCmd(cfg config, out io.Writer) *cobra.Command {
 	}
 	cmd.Flags().StringVarP(&options.appName, deploy.FlagApp, deploy.FlagAppShort, "", "The name of the app.")
 	cmd.MarkFlagRequired("app")
+	cmd.Flags().BoolVar(&options.secure, flagSecure, false, "Whether the CName should be https")
```

Comment:
This is just a nit pick, but I feel there is not much of a need to have the flag name as a const.